oak_functions/examples/weather_lookup/client/client

```diff
@@ -1,7 +1,9 @@
 #!/usr/bin/env bash
 set -o xtrace
 
+# Send the request and print only the status code and the content type.
 curl \
+  -s -o /dev/null -w "%{http_code}\n%{content_type}\n"\
```

Comment:
--silent
--output
--write-out

Reply:
I was looking for something like this. Thanks :)